src/data.js

```diff
@@ -2000,6 +2000,33 @@ module.exports = [
       'Movie Buff',
     ],
   },
+  {
+    name: 'Michael Bonner',
+    description:
+      'Full stack JavaScript and PHP developer in Salt Lake City, USA',
+    url: 'https://michaelbonner/uses',
```

Comment:
Error: Ping to "https://michaelbonner/uses" failed

Reply:
Wow... that was dumb :( Fixed now. Thanks!